ios/brave-ios/Sources/Brave/Frontend/Browser/Tab.swift

```diff
@@ -305,9 +302,10 @@ class Tab: NSObject {
   }
 
   var mimeType: String?
-  var isEditing: Bool = false
-  var shouldNotifyAdsServiceTabDidChange = true
-  var shouldNotifyAdsServiceTabContentDidChange = true
+  var isEditing = false
+  var isRestored = false
+  var isNewNavigation = true
+  var isErrorPage = false
```

Comment:
internalURL's errorPage doesn't handle 400x and 500x error codes returned from server. 
We check error codes and set `isErrorPage` to true here: https://github.com/brave/brave-core/pull/23886/files#diff-348b0ca220f2c6c15e6acd06769d042b9eb7d26355cfb397bb4bc6bab53c8c17R695

Reply:
Lets add a comment to this var stating that this is true when the resulting page is showing an error interstitial _or_ the resulting response from the web page had a 4xx or 5xx status code

Reply:
I would also prefer is these were somewhat limited to ads in scope since theres a chance these are used in the future elsewhere which may not match the exact specs of how they're used for ads

Reply:
Fixed